federatedscope/core/aggregator.py

```diff
@@ -38,9 +39,22 @@ def aggregate(self, agg_info):
         models = agg_info["client_feedback"]
         recover_fun = agg_info['recover_fun'] if (
             'recover_fun' in agg_info and self.cfg.federate.use_ss) else None
-        avg_model = self._para_weighted_avg(models, recover_fun=recover_fun)
-
-        return avg_model
+        staleness = [x[1]
```

Comment:
It seems that, in most cases, each aggregator class corresponds to a specific fl optimization algo. Thus, I am wondering is it better to make this class dedicated to FedAvg and define another specific asyncFedAvg/FedBuff/sth. else.

Reply:
I have implemented `AsynClientsAvgAggregator` for such a purpose